psi4/src/psi4/cc/cclambda/local.cc

```diff
@@ -88,14 +88,14 @@ void local_filter_T1(dpdfile2 *T1) {
     local.pairdom_len = init_int_array(nocc * nocc);
     local.pairdom_nrlen = init_int_array(nocc * nocc);
     local.eps_occ = init_array(nocc);
-    psio_read_entry(PSIF_CC_INFO, "Local Pair Domain Length", (char *)local.pairdom_len, nocc * nocc * sizeof(int));
+    psio_read_entry(PSIF_CC_INFO, "Local Pair Domain Length", (char *)local.pairdom_len, sizeof(int) * nocc * nocc);
     psio_read_entry(PSIF_CC_INFO, "Local Pair Domain NR Length", (char *)local.pairdom_nrlen,
-                    nocc * nocc * sizeof(int));
+                    sizeof(int) * nocc * nocc);
     psio_read_entry(PSIF_CC_INFO, "Local Occupied Orbital Energies", (char *)local.eps_occ, nocc * sizeof(double));
 
-    local.W = (double ***)malloc(nocc * nocc * sizeof(double **));
-    local.V = (double ***)malloc(nocc * nocc * sizeof(double **));
-    local.eps_vir = (double **)malloc(nocc * nocc * sizeof(double *));
+    local.W = (double ***)malloc(sizeof(double **) * nocc * nocc);
+    local.V = (double ***)malloc(sizeof(double **) * nocc * nocc);
+    local.eps_vir = (double **)malloc(sizeof(double) * nocc * nocc);
```

Comment:
Shouldn't this be a `sizeof(double *)` ?

Reply:
Good catch!